packages/serverpod/lib/src/database/adapters/postgres/database_connection.dart

```diff
@@ -329,33 +333,52 @@ class DatabaseConnection {
     String query, {
     int? timeoutInSeconds,
     Transaction? transaction,
-    Map<String, dynamic>? substitutionValues = const {},
+    QueryMode? queryMode,
+  }) async {
+    var result = await _query(
+      session,
+      query,
+      timeoutInSeconds: timeoutInSeconds,
+      transaction: transaction,
+      queryMode: queryMode,
+    );
+
+    return PostgresDatabaseResult(result);
+  }
+
+  Future<pg.Result> _query(
+    Session session,
+    String query, {
+    int? timeoutInSeconds,
+    Transaction? transaction,
+    QueryMode? queryMode,
   }) async {
     var postgresTransaction = _castToPostgresTransaction(transaction);
+    var timeout =
+        timeoutInSeconds != null ? Duration(seconds: timeoutInSeconds) : null;
 
     var startTime = DateTime.now();
     try {
       var context =
           postgresTransaction?.executionContext ?? _postgresConnection;
 
-      var result = await context.query(
+      var result = await context.execute(
```

Comment:
Question: Did they change the functionality of execute / query ?

Reply:
They removed query completely from the interface, only execute exists now.